src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else if ((days >= 3) && (days < 7)) {
```

Comment:
```suggestion
  }
  
  if ((days >= 3) && (days < 7)) {
```
Можно не писать else, если в предыдущем `if` есть `return`